nucleus/packager/nucleus-grizzly/pom.xml

```diff
@@ -103,6 +103,11 @@
                                     <destFileName>grizzly-npn-bootstrap.jar</destFileName>
                                 </artifactItem>
                                 <!-- Other versioned NPN Bootstrap Jars referenced in new domain.xml (since 5.184) -->
+                                  <artifactItem>
+                                    <groupId>org.glassfish.grizzly</groupId>
+                                    <artifactId>grizzly-npn-bootstrap</artifactId>
+                                    <version>1.9.payara-p1</version>
```

Comment:
Are grizzly npn changes being pushed back upstream to Grizzly. Ideally we would patch here then get it into a Grizzly release ASAP

Reply:
AFAIK, the latest version of Grizzly NPN contains support for the latest version of OpenJDK, so it's equal to our latest version of Grizzly. However, Grizzly NPN is designed to support only a single version of the JDK security classes so this fix would only be accepted after OpenJDK is updated with the same changes as the latest Zulu version. Azul claim their going to push this change to OpenJDK in the following months.